tests/test_rpm_verifier.py

```diff
@@ -130,6 +130,46 @@ def test_get_unsigned_rpms(test_input: list[str], expected: list[str]) -> None:
+            [
+                "quay.io/redhat-user-workloads/rh-acs-tenant/acs/roxctl@"
+                "sha256:3ec7a3cd38dbafafa766b22bfc3c433aaafdfed29d41ef2e01e372260b304b9d",
+                "quay.io/redhat-user-workloads/rh-acs-tenant/acs/central-db@"
+                "sha256:bedba7db0634d801be86d7d18286adfc90d251ff39be1efc567cc25d5c362c28",
+                "quay.io/redhat-user-workloads/rh-acs-tenant/acs/main@"
+                "sha256:5f715a2ca023dc549abaacb3ec4d7b4ad644c34ca8e7ff697b5cda92b811e731",
+                "quay.io/abiton1/ansible-automation-platform-2-next-eda-controller-rhel8:latest",
+            ],
+            id="snapshot test",
+        ),
+        pytest.param(
+            "quay.io/redhat-user-workloads/rh-acs-tenant/acs/roxctl@"
+            "sha256:3ec7a3cd38dbafafa766b22bfc3c433aaafdfed29d41ef2e01e372260b304b9d",
+            [
+                "quay.io/redhat-user-workloads/rh-acs-tenant/acs/roxctl@"
+                "sha256:3ec7a3cd38dbafafa766b22bfc3c433aaafdfed29d41ef2e01e372260b304b9d"
+            ],
+            id="single container",
+        ),
```

Comment:
This would have been much easier to read:
```suggestion
        pytest.param(
            "my-image",
            ["my-image"],
            id="single container",
        ),
```

Reply:
Done